src/us/kbase/common/test/service/ServiceCheckerTest.java

```diff
@@ -54,7 +54,7 @@ public static void beforeClass() throws Exception {
 		TESTS.put("short200", new TestSpec(
 				"URL " + KBASE_ENV + "/services/shock-api does not point " +
 				"to a KBase SDK generated service. Code: 200, message: OK, " +
-				"content: {\"attribute_indexes\"",
+				"content: {",
```

Comment:
Is this diff important?

Reply:
It allows the tests to pass. `attribute_indexes` is shock only, the blobstore doesn't have that field.

Reply:
:+1: